contracts/Dispatcher.sol

```diff
@@ -79,13 +79,7 @@ contract Dispatcher is IbcDispatcher, Ownable {
         AckPacket ackPacket
     );
 
-    event WriteTimeoutPacket(
-        address indexed writerPortAddress,
-        bytes32 indexed writerChannelId,
-        uint64 sequence,
-        uint64 pktTimeoutHeight,
-        uint64 pktTimeoutTimestamp
-    );
+    event WriteTimeoutPacket(address indexed writerPortAddress, bytes32 indexed writerChannelId, uint64 sequence);
```

Comment:
They are used in the correct flow.   So please don't remove them. 

- pktTimeoutHeight / Timestamp are stored on polymer chain in a specific block.   `eth2-relayer` is able to scan those if needed, but it will be a harder work, and need to look very deep (because no clue about where the block is, but this is a timeout, so the block might be far lower than current height)

- So the current flow is letting ETH side provide this data directly.

Reply:
These fields are part of the original packet. So relayers don't need them to id packets

Reply:
Just saw @qinlz2's comments. If it's still used, I can bring them back. 
IMO we should delete them later